tutorials/matchmaker101/matchmaker.yaml

```diff
@@ -0,0 +1,151 @@
+# Copyright 2019 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# This file contains the sample minimal pod definitions for all of the components that one may need to use Open Match as a match maker.
+# You can find the same pod definitions within the sub-folders under the /tutorials/ directory
+# Run `kubectl apply -f matchmaker.yaml` to deploy these definitions.
+
+apiVersion: v1
+kind: Pod
+metadata:
+  name: mm101-tutorial-director
+  namespace: mm101-tutorial
+spec:
+  containers:
+  - name: mm101-tutorial-director
+    image: REGISTRY_PLACEHOLDER/mm101-tutorial-director:latest
+    imagePullPolicy: Always
+  hostname: mm101-tutorial-director
+---
+apiVersion: v1
+kind: Pod
+metadata:
+  name: mm101-tutorial-frontend
+  namespace: mm101-tutorial
+spec:
+  containers:
+  - name: mm101-tutorial-frontend
+    image: REGISTRY_PLACEHOLDER/mm101-tutorial-frontend:latest
+    imagePullPolicy: Always
+  hostname: mm101-tutorial-frontend
+---
+kind: Service
+apiVersion: v1
+metadata:
+  name: mm101-tutorial-matchfunction
+  namespace: mm101-tutorial
+  labels:
+    app: mm101-tutorial
+    component: matchfunction
+spec:
+  selector:
+    app: mm101-tutorial
+    component: matchfunction
+  clusterIP: None
+  type: ClusterIP
+  ports:
+  - name: grpc
+    protocol: TCP
+    port: 50502
+---
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: mm101-tutorial-matchfunction
+  namespace: mm101-tutorial
+  labels:
+    app: mm101-tutorial
+    component: matchfunction
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: mm101-tutorial
+      component: matchfunction
+  template:
+    metadata:
+      namespace: mm101-tutorial
+      labels:
+        app: mm101-tutorial
+        component: matchfunction
+    spec:
+      containers:
+      - name: mm101-tutorial-matchfunction
+        image: REGISTRY_PLACEHOLDER/mm101-tutorial-matchfunction:latest
+        imagePullPolicy: Always
+        ports:
+        - name: grpc
+          containerPort: 50502
+---
+kind: Service
+apiVersion: v1
+metadata:
+  name: om-evaluator
+  namespace: open-match
+  labels:
+    app: mm101-tutorial
+    component: evaluator
+spec:
+  selector:
+    app: mm101-tutorial
+    component: evaluator
+  clusterIP: None
+  type: ClusterIP
+  ports:
+  - name: grpc
+    protocol: TCP
+    port: 50508
+---
+apiVersion: autoscaling/v1
```

Comment:
The PodAutoscaler is one of the production best practices. We could potentially skip this part to keep things easy to follow.

Reply:
Removed! Thanks.